mcpgrafana.go

```diff
@@ -56,6 +58,17 @@ type GrafanaConfig struct {
 	// Debug enables debug mode for the Grafana client.
 	Debug bool
 
+	// EnableTracing enables OpenTelemetry instrumentation for both tool execution
+	// and HTTP requests, providing complete observability with perfect correlation
+	// between tool spans and HTTP request spans. Defaults to false for minimal overhead.
+	EnableTracing bool
```

Comment:
I wonder if we should just get rid of this flag and always add the trace context propagation by modifying the Grafana client transport. Is there a reason we don't want to do that? 🤔